### ISSUE TITLE
Detailed commit message

### ISSUE THREAD

**Issue by b3u** [GitJournal/GitJournal#201]
## Environment

**App version:** 1.62.0+1553
**Device information:**
 - OS name and version: Android 8.1.0
 - Manufacturer: LG
 - Model: Phoenix 4

## Description

**What you'd like to happen:** 
Commit messages should have more details, so they are distinguishable.
**Example**
```txt
Edited ${Multiple notes,"note-title"}

11:03pm on August 2, 2020
via GitJournal
```
**Alternatives you've considered:**   
Possibly add option to toggle certain details.

**Comment by vHanda** [comment]
This is very simple to do. Are you sure you want the time in it? The commit already contains the time.

I like the "Edited via GitJournal" part.

**Comment by b3u** [comment]
> Are you sure you want the time in it? The commit already contains the time.

@vHanda Good point. I was just trying to think of easily accessible data which can distinguish commits. The goal here is that each commit should look somewhat different.

Another option is to specify if metadata was added, and what the fields were. Also, if the user edits more than one note, the note titles maybe should be listed in the commit body.

**Comment by vHanda** [comment]
Makes sense.

This sounds like a fun side project, generating useful commit messages out of a before and an after. I'll add some basic heuristics, and a few tests, and then open it up to more suggestions.

**Comment by b3u** [comment]
@vHanda I think `via gitjournal` could get really annoying, unless it has a toggle option. So I guess I'm really asking for the filename in the commit title, or in the body if there are multiple.
